### PR TITLE
Small fixes for GLK/CML kwd topologies

### PR DESCRIPTION
**Pull request by ranj063**

### CODE REVIEW

**Comment by ranj063** [comment]
@fredoh9 @xiulipan this should fix the compilation errors seen on all topology builds in CI

File: tools/topology/sof-cml-rt5682-kwd.m4 [hunk 1]
```diff
@@ -198,7 +198,7 @@ PIPELINE_ADD(sof/pipe-detect.m4,
 	16000, 16000, 16000)
 
 # Connect pipelines together
-SectionGraph."pipe-sof-apl-keyword-detect" {
+SectionGraph."pipe-cml-keyword-detect" {
```
Comment (juimonen) [inline tools/topology/sof-cml-rt5682-kwd.m4]: so this should not be "pipe-sof-cml-keyword-detect" ? or does it matter?

Comment (ranj063) [reply tools/topology/sof-cml-rt5682-kwd.m4]: @juimonen it doesnt matter but I fixed it already :d

Comment (keyonjie) [reply tools/topology/sof-cml-rt5682-kwd.m4]: The naming here is not important, but yes, making it be consistent with the file name makes sense to me.

**Comment by xiulipan** [comment]
@ranj063 Thanks for help!

**Pull request merged**